src/libsystemd/libsystemd.sym

```diff
@@ -819,4 +819,8 @@ global:
         sd_session_get_start_time;
         sd_uid_get_login_time;
         sd_pid_notifyf_with_fds;
+        sd_event_add_memory_pressure;
+        sd_event_source_set_memory_pressure_type;
+        sd_event_source_set_memory_pressure_period;
+        sd_event_trim_memory;
```

Comment:
By the way, why these missing symbols are not detected by CIs ?? @mrc0mmand

Reply:
I only discovered them by chance in #27442, definitely seems to be something missing from our coverage. Maybe we could write some unit test for this?

Reply:
I thought we had a test about such, but not sure, maybe my memory is broken.

Reply:
Ah, I mean, test-libsystemd-sym.